monitoring/collect_and_push_job_stats.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import argparse
+import getpass
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from stats import collect_and_push_job_stats
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```

Comment:
Maybe create some `MysqlCommonArgumentParser` for all `--mysql-*` arguments. See an example [here](https://github.com/codilime/contrail-windows-ci/blob/development/ansible/files/vmware_api.py#L16).